bigstream/transform.py

```diff
@@ -277,7 +287,7 @@ def compose_displacement_vector_fields(
             second_field[..., iii], first_field[..., iii],
             second_spacing, first_spacing,
             transform_list=[second_field,],
-            **kwargs,
+            extrapolate_with_nn=extrapolate_with_nn,
```

Comment:
IMO it is much cleaner to spell out the exact parameter needed - considering that `extrapolate_with_nn` is the only one passed at this point